### PR TITLE
Portuguese translations Part7-8-8bis

### PR DESCRIPTION
**Pull request by joaolcaas**
Related to #2776 

- [x]  Part 01 - The Basic Tools of Private Deep Learning.ipynb
- [x]  Part 02 - Intro to Federated Learning.ipynb
- [x]  Part 03 - Advanced Remote Execution Tools.ipynb
- [x] Part 04 - Federated Learning via Trusted Aggregator.ipynb
- [x] Part 05 - Welcome to the Sandbox.ipynb
- [x]  Part 06 - Federated Learning on MNIST using a CNN.ipynb
- [x] Part 07 - Federated Learning with Federated Dataset.ipynb
- [x] Part 08 - Introduction to Plans.ipynb
- [x] Part 08 bis - Introduction to Protocols.ipynb
- [x] Part 09 - Intro to Encrypted Programs.ipynb
- [x] Part 10 - Federated Learning with Secure Aggregation.ipynb
- [x] Part 11 - Secure Deep Learning Classification.ipynb
- [ ] Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb
- [ ] Part 12 bis - Encrypted Training on MNIST.ipynb
- [ ] Part 13a - Secure Classification with Syft Keras and TFE - Public Training.ipynb
- [ ] Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb
- [ ] Part 13c - Secure Classification with Syft Keras and TFE - Private Prediction Client.ipynb

### CODE REVIEW

File: examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb [hunk 1]
```diff
@@ -0,0 +1,217 @@
+{
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb]: <p>Just a suggestion. I don't think <em>"Federado"</em> is in agreement with <em>"Base"</em>. A better alternative is <em>"Base de Dados Federada"</em> or <em>"Conjunto de Dados Federado"</em>.</p><p> </p><blockquote>Em português, federado não está nem em concordância com <em>Base</em> nem com <em>Dados. </em>Se estiver se referindo à base, Base Federada, se aos dados, Base de Dados federados. Poderia ser também Conjunto de Dados Federado, ou Dataset federado.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb]: <p>I think those two comments were missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb]: <p>treinamento*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb [hunk 2]
```diff
@@ -0,0 +1,648 @@
+{
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb]: <blockquote>&nbsp;... Em particular, isso <strong>exclúi </strong>estruturas lógicas [...]</blockquote><p>exclúi -&gt; exclui</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb]: <blockquote># hook PyTorch ie adiciona funcionalidades extras&nbsp;</blockquote><p>Suggestion: use <em>"i.e.".</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb]: <blockquote>&nbsp;definimos&nbsp;<em>workers</em>&nbsp;remotos ou&nbsp;<strong><em>devices</em>&nbsp;,</strong></blockquote><p><em>"devices</em>&nbsp;," -&gt; "devices,"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb]: <blockquote>visto <strong>ateriormente</strong></blockquote><p>ateriormente -&gt; anteriormente</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

File: examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb (hunk 1 above)
Comment (joaolcaas) [reply examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Base de Dados Federado.ipynb]: Indeed!

File: examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Conjunto de Dados Federado.ipynb [hunk 3]
```diff
@@ -92,13 +92,13 @@
...
    "source": [
-    "# Cast the result in BaseDatasets\n",
+    "# Carrega o resultado em BaseDatasets\n",
     "datasets = []\n",
     "for worker in boston_data.keys():\n",
     "    dataset = sy.BaseDataset(boston_data[worker][0], boston_target[worker][0])\n",
     "    datasets.append(dataset)\n",
     "\n",
-    "# Build the FederatedDataset object\n",
+    "# Contrura o objeto FederatedDataset\n",
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 07 - Aprendizado Federado com Conjunto de Dados Federado.ipynb]: Contrura -> Construa

File: examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb [hunk 4]
```diff
@@ -71,7 +71,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Para ser consistente com o conceito fornecido no artigo referenciado, definimos _workers_ remotos ou _devices_ , provendo eles com algum dado. "
+    "Para ser consistente com o conceito fornecido no artigo referenciado, definimos _workers_ remotos ou devices , provendo eles com algum dado. "
```
Comment (hericlesme) [inline examples/tutorials/translations/português/Parte 08 - Introdução a Planos.ipynb]: Remove the space between "devices" and ",".

**Pull request merged**